packages/aws-cdk-lib/aws-appsync/lib/graphqlapi.ts

```diff
@@ -186,15 +186,23 @@ export interface AuthorizationConfig {
  */
 export enum FieldLogLevel {
   /**
-   * No logging
+   * Resolver logging is disabled
```

Comment:
Updated the descriptions for existing levels (NONE, ERROR, and ALL) to be more user-friendly, aligning them with the AWS Management Console.

![image](https://github.com/user-attachments/assets/baba13e2-b618-4198-81d5-4477f48defd1)